application/controllers/admin/solutions.php

```diff
@@ -1103,10 +1116,12 @@ public function get_valuation_table(): void
         $filter = $this->input->post('filter');
         $this->store_valuation_tables_filter($filter);
         $this->inject_stored_valuation_tables_filter();
+
+        $table_data = $this->get_valuation_table_data((int)@$filter['course'], @$filter['group'], (bool)@$filter['simple']);
+        $this->add_virtual_task_set_types_data($table_data, (int)@$filter['course']);
         
         $this->parser->assign(
-            'table_data',
-            $this->get_valuation_table_data((int)@$filter['course'], @$filter['group'], (bool)@$filter['simple'])
+            'table_data', $table_data
         );

```

Comment:
toto uz moze byt na jednom riadku

```suggestion
        $this->parser->assign('table_data', $table_data);
```